### ISSUE TITLE
Multiarch images

### ISSUE THREAD

**Issue by agorgl** [fonoster/routr#141]
Thank your for the amazing project!
It would be awesome if you could provide multi-arch images specifically for armv7/arm64 variants that could work on ARM clusters
(e.g. a k3s cluster running on a fleet of rpi's)

## Funding
* You can sponsor this specific effort via a [Polar.sh](https://polar.sh) pledge below
* We receive the pledge once the issue is completed & verified

<a href="https://polar.sh/fonoster/routr/issues/141">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/fonoster/routr/issues/141/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/fonoster/routr/issues/141/pledge.svg">
</picture>
</a>

**Comment by psanders** [comment]
@agorgl 

Thanks a lot for raising this issue. 

`Rourt v1` will soon enter "maintenance" mode, and we will not add any new capabilities or features.

We hope to announce the next release soon; this capability could make a good addition.

I will leave this open for now. Can we revisit this in about four weeks? 

(We also accept PRs)

**Comment by KrisMie** [comment]
I am also looking for arm64 support. Additionally due to Raspberry Pi shortages I moved to Debian 10 powered Banana Pi M-5. `fonoster/routr:1.2.0` image doesn't seem to be working here.
Here's my Arch and OS:
```
Linux bananapi 4.9.241-BPI-M5 #2 SMP PREEMPT Mon Jun 21 16:29:40 HKT 2021 aarch64 GNU/Linux

Distributor ID: Debian
Description:    Debian GNU/Linux 10 (buster)
Release:        10
Codename:       buster
```

When running the image with command: 
```
docker run -it \
    -p 4567:4567 \
    -p 5060:5060 \
    -p 5060:5060/udp \
    -e EXTERN_ADDR=192.168.1.100 \
    -e DATA_SOURCE_PROVIDER=files_data_provider \
    -v $(pwd)/config:/opt/routr/config \
    fonoster/routr
```

I get this error:

`exec /tini: exec format error`

Is this due to the fact its BPI? Can anyone confirm this image actually works on arm64 RPI as I currently don't have one to test.

**Comment by psanders** [comment]
@KrisMie I feel like we are missing something important. I was able to run Routr 1.2.x on the new Macs. I'm thinking on setting up a Pi and do some testing, since it looks like more and more people are interested on this.

**Comment by kanimaru** [comment]
For K8s it seems that there is the ARM64 Image missing for requester and edgeport what has to be done to support ARM64 here?

**Comment by psanders** [comment]
@kanimaru, if I recall correctly, the base image doesn't support ARM. We will need to research and test using other images.

**Comment by psanders** [comment]
This issue was solved in [v2.6.0](https://github.com/fonoster/routr/releases/tag/v2.6.0).
